### PR TITLE
[Fix](Txn) Fix wrong columns sequence in txn model

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run p0

**Comment by Yukang-Lian** [comment]
run p0

**Comment by Yukang-Lian** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalSink.java [hunk 1]
```diff
@@ -54,4 +54,8 @@ public List<Slot> computeOutput() {
                 .map(NamedExpression::toSlot)
                 .collect(ImmutableList.toImmutableList());
     }
+
+    public List<NamedExpression> getOutputExprs() {
+        return outputExprs;
+    }
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalSink.java:60]: remove override function in `org.apache.doris.nereids.trees.plans.physical.PhysicalResultSink`

**Review by morrySnow — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**